app/helpers/record_helper.rb

```diff
@@ -75,4 +81,13 @@ def should_show_copy_recommendations_button?(record)
   def copy_recommendations_active_class(record)
     record.copy_recommendations_note.nil? ? 'eui-btn--green' : 'eui-btn--disabled'
   end
+
+  # When a record has multiples, Dashboard stores them in the value field
+  # formatted like: "• <value1>\n• <value2>\n ..."
+  # Returns an array like: [value1, value2, ...]
+  def clean_up_campaign(campaign)
+    return [] if campaign.nil?
+
+    campaign.tr('•', '').split("\n").each(&:strip!)
```

Comment:
`["Aqua", "LANDSAT-7", "", "TERRA", "Terra", "• Terra\n• Aqua", "OV-105", "SUOMI-NPP", "• GLONASS\n• GPS", "• Beidou\n• Galileo\n• GLONASS\n• GPS\n• GROUND STATIONS\n• QZSS\n• IRNSS", "• Aqua\n• Terra", "In Situ Ocean-based Platforms", "Suomi-NPP", "STS-99", "NOAA-20", "ISS", "• Terra\n• NOAA-7\n• NOAA-9\n• NOAA-11\n• NOAA-14", "• Terra\n• NOAA-7\n• NOAA-9\n• NOAA-11\n• NOAA-14\n• LANDSAT-5\n• LANDSAT-7", "• LANDSAT-7\n• DEM\n• LANDSAT-5", "• LANDSAT-8\n• SENTINEL-2\n• DEM\n• LANDSAT-5", "• LANDSAT-8\n• LANDSAT-7\n• DEM\n• LANDSAT-5", "• LANDSAT-8\n• LANDSAT-7\n• DEM", "• LANDSAT-8\n• LANDSAT-7\n• LANDSAT-5\n• SENTINEL-2", "• LANDSAT-7\n• LANDSAT-5\n• Terra", "• NOAA-7\n• NOAA-9\n• NOAA-11\n• NOAA-14\n• Terra", "EO-1", "• LANDSAT-7\n• LANDSAT-5", "Sentinel-3A", "Sentinel-3B", "G-IV", "AC-680E"]`
That is a list of platform short names being stored in dashboard.  Each string is displayed as a list in the review section.  Campaigns (and every other multi item I've looked at, I chose platforms at a whim here) are stored the same way.  
I am not removing it from the column which is being used to display data.  I am adding a new column (to a different table) and reformatting the data so that it is more programmatically usable.  Digging deeper into it, the bullet is being added as part of ingest and is character `bullet = "\u{2022} "`.